apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -70,6 +70,24 @@ function renderSectionsTable(sections) {
     </Provider>,
     element
   );
+
+  initGoogleClassroom();
```

Comment:
Is this the right place for this call? `renderSectionsTable` is only called if we have an experiment on. (This may well be the right place, just making sure).